components/core/DataView.js

```diff
@@ -228,6 +198,8 @@ export default class DataView extends React.Component {
     startIndex: 0,
```

Comment:
With your new changes, everywhere in the code where you see startIndex or this.state.startIndex you can remove it. Treat it like 0 now b/c it's always 0 (so if you see checked[this.state.startIndex + i], that turns into checked[i]). It was only necessary for when we had pagination and didn't display images starting from 0 (e.g. if you're on page 2, it would show images 20-40). Make sure you test delete and add to slate after you make this change

Reply:
ok, so after implementing this in a3fbbe3 I get the following memory leak error and reload issue.

```
Warning: Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in the componentWillUnmount method.
    in DataView (created by SceneHome)
    in div (created by SceneHome)
    in div (created by Context.Consumer)
    in EmotionCssPropInternal (created by ScenePage)
    in div (created by Context.Consumer)
    in EmotionCssPropInternal (created by ScenePage)
    in ScenePage (created by SceneHome)
    in SceneHome
```

Reply:
It happens when I reach the bottom of the page, and then after all image load successfully I get kicked to the top of the page, and then this error happens. I thought it was because of the `_handleScroll` function but haven't found the issue the issue yet.